docs/user_guide/HOWTO/NFS.rst

```diff
@@ -15,20 +15,60 @@ and ``pupmod-simp-simp``.
 Known Issues
 ------------
 
-The ``autofs`` package that was released with CentOS 6.8 (**autofs-5.0.5-122**) worked
-properly over a stunnel connection.
+Stunnel and Autofs
+^^^^^^^^^^^^^^^^^^
+The ``autofs`` packages that were released with CentOS 6.8 (**autofs-5.0.5-122**)
+and CentOS 7.3 (**autofs-5.0.7-56**) worked properly over a stunnel connection.
 
-The release shipped with CentOS 6.9 (**5.0.5-132**) prevents any connection from happening
-to the local stunnel process and breaks mounts to remote systems over stunnel connections.
+The release shipped with CentOS 6.9 (**5.0.5-132**)  and with CentOS 7.4 (**5.0.7-69**)
+prevents any connection from happening to the local stunnel process and breaks mounts to
+remote systems over stunnel connections.
 
 To use NFS over stunnel and automount directories the old package must be used.
+To determine what version of autofs is installed, run ``automount -V``.
 
-To determine what package is installed on the system, run ``automount -V``.
+To force the package to the version wanted despite the fact that a newer version is available:
 
-This does not effect CentOS 7.
+First make sure the package is available via your package-management facility then
+set the package version in hiera:
 
-This has been identified as a bug in autofs and is being publicly
-tracked at https://bugs.centos.org/view.php?id=13575.
+In CentOS 7.4:
+
+.. code-block:: puppet
+
+  ---
+  autofs::autofs_package_ensure:  '5.0.7-56.el7'
```

Comment:
Where can users find these packages? Should we mention that?

Reply:
Yeah they're in the Vault repo for CentOS versions X.Y-1, which is not clear. 

http://vault.centos.org/7.3.1611/os/x86_64/Packages/autofs-5.0.7-56.el7.x86_64.rpm
http://vault.centos.org/6.8/os/x86_64/Packages/autofs-5.0.5-122.el6.x86_64.rpm